template/oss-npm-package/.nvmrc

```diff
@@ -1 +1 @@
-18
```

Comment:
I've left the package.json engines alone in the packages as that would force consumers to update to node 20 which doesn't quite seem right?